openstack/networking/v2/extensions/portsbinding/requests.go

```diff
@@ -0,0 +1,104 @@
+package portsbinding
+
+import (
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack/networking/v2/ports"
+)
+
+type CreateOpts struct {
+	Parent   ports.CreateOpts
```

Comment:
Let's follow the example here: https://github.com/rackspace/gophercloud/blob/master/openstack/compute/v2/extensions/bootfromvolume/requests.go#L47

`CreateOptsExt` and an unnamed field for the parent `CreateOpts`